examples/protyping/pymedphys-segmentation/data_generator-bf.py

```diff
@@ -0,0 +1,249 @@
+# %load generator
```

Comment:
When you're writing these scripts, now would be a good time to put the Apache copyright header in. If I've contributed a significant section of the code in that particular file make sure to add your name and mine. Put your name first though.